src/OpenRiaServices.Tools/Test/ClientCodeGenerationDispatcherTests.cs

```diff
@@ -555,6 +555,15 @@ public void ClientCodeGenerationDispatcher_Error_TypeLoadException()
 
                 Assert.IsTrue(File.Exists(typeLoadExceptionProjectLocation), "Expected TypeLoadExceptionProject.dll to coreside with this assembly in test folder");
 
+                using (var assemblyDefinition = AssemblyDefinition.ReadAssembly(typeLoadExceptionProjectLocation, new ReaderParameters { ReadWrite = true }))
```

Comment:
Since the assembly is modified from here now, should we instead create the assembly that generates the TypeLoadException?

Reply:
I sounds like a good idea if it is not to difficult to remove the TypeLoadException project and just create the assembly directyle.

Reply:
I will leave it as it is for now